deployments/helm/nvidia-device-plugin/values.yaml

```diff
@@ -35,6 +35,8 @@ deviceIDStrategy: null
 nvidiaDriverRoot: null
 gdsEnabled: null
 mofedEnabled: null
+imex:
+  nodesConfigFile: null
```

Comment:
I don't think this should be configurable from helm. Making it configurable from the config file should be enough.

Reply:
@ArangoGutierrez did we decide to also set this in Helm?

Reply:
I thought having a way to define an alternative path for the imex config file was a good thing using Helm, but not a strong opinion

Reply:
Please drop this.

Reply:
Note that if we allow this to be specified, then the mount path on the host should probably be dependent on this value. Could we add documentation to this option here so as to better explain how the various values are handled.

For example, this is the path in the container and not on the host. How is that handled?

Reply:
We could make the host path for this configurable in the future. For now we should just hardcode it as `{{hostDriverPath}}/etc/nvidia-imex`. The container path should always (now and in the future) be `/etc/nvidia-imex`.

Reply:
Meaning this option should disappear and we should hard-code this in the templates.